src/main/java/bio/terra/service/snapshot/flight/export/SnapshotExportUtils.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.service.snapshot.flight.export;
+
+import bio.terra.stairway.FlightContext;
+
+public class SnapshotExportUtils {
+
+  public static String getFileName(FlightContext context) {
+    return String.format("%s_export_gs_path_mapping/gs_path_mapping.json", context.getFlightId());
```

Comment:
Just want to check: are there limitations to what can be used in a file name? Flight Ids can contain dashes and underscores (and I believe can start with them). Just want to make sure that's ok

Reply:
Though I guess we already do this

Reply:
ah ok cool